plugin/package.json

```diff
@@ -2,13 +2,15 @@
   "devDependencies": {
     "eslint": "^8.35.0",
     "prettier": "^2.5.1",
-    "typescript": "^4.1.3"
+    "typescript": "^4.1.3",
+    "esbuild": "^0.17.11"
   },
   "license": "MIT",
   "readmeFilename": "README.md",
   "scripts": {
     "prepare": "yarn generate",
-    "generate": "yarn tsc",
+    "generate": "yarn tsc && yarn bundle",
+    "bundle": "yarn esbuild src/plugin.js --bundle --external:@babel --external:convert-source-map --external:fs --log-level=warning --platform=node --sourcemap=linked --outfile=index.js && rm src/*.js src/*.js.map",
```

Comment:
Can we move `rm src/*.js src/*.js.map` to a separate `clean` script and call it inside `generate`?